Vizzuality/half-earth-v3
# Develop - active layer

## Merge Develop change to set active layer in biodiversity side bar control
### Description
Biodiversity dropdown is not defaulting to an active layer option when loading from a URL, this fixes that issue

### Testing instructions
Select a layer from one of the Biodiversity layer dropdowns, then open the url in a new tab and see that the dropdown option is selected
